### PR TITLE
Documentation for decorator arg

### PR DESCRIPTION
**Pull request by hackaugusto**
This adds a bit of documentation to `arg`, the intention is to make it a bit easier to understand it's usage.

### CODE REVIEW

File: aiven/client/cli.py [hunk 1]
```diff
@@ -31,9 +31,7 @@
 PLUGINS = []
 
 try:
-    from aiven.admin import (
-        plugin as adminplugin,
-    )  # pylint: disable=import-error,no-name-in-module
+    from aiven.admin import plugin as adminplugin  # pylint: disable=import-error,no-name-in-module
```
Comment (rominf) [inline aiven/client/cli.py:34]: Unrelated change?

Comment (hackaugusto) [reply aiven/client/cli.py:34]: My setup runs `yapf` automatically. Should I revert it?

Comment (rominf) [reply aiven/client/cli.py:34]: Well, I think it's not that important, moreover, if it's a `yapf` related change. I'll merge this as is :)

**Review by rominf — APPROVED** [review]

File: aiven/client/cli.py (hunk 1 above)
Comment (hackaugusto) [reply aiven/client/cli.py:34]: thanks!
